### ISSUE TITLE
<mat-slider> disable value change on click rage line 

### ISSUE THREAD

**Issue by webdevlapani** [angular/components#15526]
#### Please describe the feature you would like to request.
- we need to one property using this we can control the range value change or not on click or touch on `<mat-slider>` line. 

#### What is the use-case or motivation for this proposal?
I use material angular in my website its work good but when I use in a mobile device I face a problem like when I scroll down that time I touch by mistake on mat range slider reason for that my rage was change if I not want

So I want a solution for that like I need only mat-slider value will change when user drag handle of mat-slider instead of clicking slider.

#### Is there anything else we should know?
https://stackoverflow.com/questions/55205407/mat-slider-touch-issue-in-mobile-device

**Comment by Domvel** [comment]
The change event is fired twice. See https://github.com/angular/material2/issues/14363 It's a bug. 
If you click on the "range line" the event should also be fired only once. Only on pointer (mouse / touch) up.

**Comment by webdevlapani** [comment]
> The change event is fired twice. See #14363 It's a bug.
> If you click on the "range line" the event should also be fired only once. Only on pointer (mouse / touch) up. But your request to disable click on range-line is good. In other words: Only allow slider-thumb interaction.

Yes that's really good.

**Comment by webdevlapani** [comment]
@jelbourn This is important for the community because in the mobile device when we scroll that time changed the value of the range slider if we don't want

**Comment by igornes** [comment]
Agree that this is an important request. We have the same issue with phone interactions. Users set the value of the slider by mistake while swiping vertically to scroll the form.

**Comment by BrianMikinski** [comment]
Any idea of a work around or if the Angular team will give us an API to do accomplish this?

**Comment by jelbourn** [comment]
Duplicate of #4711
